src/client/app/containers/app.ts

```diff
@@ -23,7 +24,7 @@ import { isNil } from 'lodash';
                     <command icon="PageCheckedin" title="${Strings.shareMenuPublic}" (click)="shareGist({isPublic: true, isUpdate: false})"></command>
                     <command icon="ProtectedDocument" title="${Strings.shareMenuPrivate}" (click)="shareGist({isPublic: false, isUpdate: false})"></command>
                     <command id="CopyToClipboard" icon="Copy" title="${Strings.shareMenuClipboard}" (click)="shareCopy()"></command>
-                    <command icon="Download" title="${Strings.shareMenuExport}" (click)="shareExport()"></command>
+                    <command [hidden]="!isDownloadZipSupported" icon="Download" title="${Strings.shareMenuExport}" (click)="shareExport()"></command>
```

Comment:
I'm not in love with hiding the button.  The developer has no idea why it's not available, or they might watch a video and see it working for someone, but then not see it themselves.

I would prefer to still keep the button, but then fail with an error message saying that export on Mac unfortunately isn't supported; but that the user can export from a non-Mac machine if they wish.